web/src/locales/en.json

```diff
@@ -291,6 +291,7 @@
     "changeTheme": "Change theme",
     "noParserInfo": "No data available",
     "temporaryDataOutage": "Live data temporarily unavailable",
+    "temporarilyUnavailable": "Temporarily unavailable",
```

Comment:
I'm thinking that maybe we should say we are using fallbacks instead?
That way it makes sense that the carbon intensity is still visible in the details of the importing country?

(Just a thought on my side though, but it might be even more confusing).

Reply:
I see your point, but not sure how we could phrase it in a very short, yet easily understandable way :sweat_smile: